ee/debug/checkups/power_windows.go

```diff
@@ -35,67 +37,50 @@ func (p *powerCheckup) Run(ctx context.Context, extraWriter io.Writer) error {
 		return fmt.Errorf("running powercfg.exe: error %w, output %s", err, string(out))
 	}
 
-	sprHandle, err := os.Open(tmpFilePath)
-	if err != nil {
-		return fmt.Errorf("opening system power report: %w", err)
-	}
-	defer sprHandle.Close()
-
 	extraZip := zip.NewWriter(extraWriter)
 	defer extraZip.Close()
 
-	zippedPowerReport, err := extraZip.Create("power.html")
-	if err != nil {
-		return fmt.Errorf("creating power report zip file: %w", err)
-	}
-
-	if _, err := io.Copy(zippedPowerReport, sprHandle); err != nil {
-		return fmt.Errorf("copying system power report: %w", err)
+	// Add the power report using addFileToZip
+	if err := addFileToZip(extraZip, tmpFilePath); err != nil {
+		return fmt.Errorf("adding power report to zip: %w", err)
 	}
 
+	// Get available sleep states
 	powerCfgSleepStatesCmd, err := allowedcmd.Powercfg(ctx, "/availablesleepstates")
 	if err != nil {
 		return fmt.Errorf("creating powercfg sleep states command: %w", err)
 	}
 
 	hideWindow(powerCfgSleepStatesCmd)
-	availableSleepStatesOutput, err := powerCfgSleepStatesCmd.CombinedOutput()
+	sleepStatesOutput, err := powerCfgSleepStatesCmd.Output()
 	if err != nil {
-		return fmt.Errorf("running powercfg.exe for sleep states: error %w, output %s", err, string(availableSleepStatesOutput))
+		return fmt.Errorf("running powercfg.exe for sleep states: error %w", err)
```

Comment:
We include the `output %s` in the error because it's the only way we get useful error information on Windows -- why remove it? Same q for line 74 below

Reply:
added back outputs